plugins/PagePerformance/Reports/Get.php

```diff
@@ -95,4 +98,11 @@ private function addSparklineColumns(Sparklines $view)
             $view->config->addSparklineMetric([$metric], $count++);
         }
     }
+
+    private function configureFooterMessage(ViewDataTable $view)
+    {
+        $out = '';
+        EventDispatcher::getInstance()->postEvent('Template.afterSeoWebVitals', array(&$out));
```

Comment:
For consistency the event should be named after where it is placed, not what it will place. 
```suggestion
        EventDispatcher::getInstance()->postEvent('Template.afterPagePerformanceReport', array(&$out));
```

Reply:
Did you consider that it is called in 2 places? I actually named it like this originally:
https://github.com/matomo-org/matomo/pull/17979/commits/4d8842843e00479279968a92f6cdbbbdc5123166#diff-e4961e62956cdd65b1454b740d30229a06b39429ddd8ad7fb257fdf1d094a1fbR34

Should I rename it back, even though it is in 2 places and therefore not clear what the name should be?

I interpreted this as the way I ended up doing it:
https://developer.matomo.org/guides/events#event-naming-conventions

Reply:
That be good to rename it like originally. It's fine if it is used in two places we can just listen to both events in the plugin. It's just we might never know what it's used for so we name the events where they are placed basically.